src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -78,41 +78,43 @@ enum CORINFO_InstructionSet
     InstructionSet_AVX10v1=33,
     InstructionSet_AVX10v1_V256=34,
     InstructionSet_AVX10v1_V512=35,
-    InstructionSet_VectorT128=36,
-    InstructionSet_VectorT256=37,
-    InstructionSet_VectorT512=38,
-    InstructionSet_X86Base_X64=39,
-    InstructionSet_SSE_X64=40,
-    InstructionSet_SSE2_X64=41,
-    InstructionSet_SSE3_X64=42,
-    InstructionSet_SSSE3_X64=43,
-    InstructionSet_SSE41_X64=44,
-    InstructionSet_SSE42_X64=45,
-    InstructionSet_AVX_X64=46,
-    InstructionSet_AVX2_X64=47,
-    InstructionSet_AES_X64=48,
-    InstructionSet_BMI1_X64=49,
-    InstructionSet_BMI2_X64=50,
-    InstructionSet_FMA_X64=51,
-    InstructionSet_LZCNT_X64=52,
-    InstructionSet_PCLMULQDQ_X64=53,
-    InstructionSet_POPCNT_X64=54,
-    InstructionSet_AVXVNNI_X64=55,
-    InstructionSet_MOVBE_X64=56,
-    InstructionSet_X86Serialize_X64=57,
-    InstructionSet_AVX512F_X64=58,
-    InstructionSet_AVX512F_VL_X64=59,
-    InstructionSet_AVX512BW_X64=60,
-    InstructionSet_AVX512BW_VL_X64=61,
-    InstructionSet_AVX512CD_X64=62,
-    InstructionSet_AVX512CD_VL_X64=63,
-    InstructionSet_AVX512DQ_X64=64,
-    InstructionSet_AVX512DQ_VL_X64=65,
-    InstructionSet_AVX512VBMI_X64=66,
-    InstructionSet_AVX512VBMI_VL_X64=67,
-    InstructionSet_AVX10v1_X64=68,
-    InstructionSet_AVX10v1_V256_X64=69,
-    InstructionSet_AVX10v1_V512_X64=70,
+    InstructionSet_APX=36,
```

Comment:
Does APX support 32bit at all?

> Intel®Advanced Performance Extensions (Intel® APX) expands the Intel® 64 instruction set

or is everything just generated with both x86 and x64?

Reply:
Thanks for bringing this up,

No, APX features is available only under 64-bit mode
Reference: https://www.intel.com/content/www/us/en/content-details/819797/intel-advanced-performance-extensions-intel-apx-architecture-specification.html
>3.1.4 Intel® APX features are only available in IA-32e 64-bit Protected Mode, and are an XSAVE-enabled feature
which requires XCR0 enabling before using the new Intel® APX ISA, new Intel® APX prefixes (REX2) and
prefix extensions (EVEX extensions). See section 3.1.4.2 for details on XCR0-enabling for Intel® APX

Based on this fact, I understand APX could be different from existing ISAs and we are open to take suggestions on this part.